libs/datasets/sources/nytimes_dataset.py

```diff
@@ -52,5 +52,6 @@ def standardize_data(cls, data: pd.DataFrame) -> pd.DataFrame:
         # Super hacky way of filling in new york.
         data.loc[data[cls.Fields.COUNTY] == "New York City", "county"] = "New York County"
         data.loc[data[cls.Fields.COUNTY] == "New York County", "fips"] = "36061"
+        data.loc[data[cls.Fields.COUNTY] == "Unknown", "fips"] = "99999"
```

Comment:
What happens if you instead drop these rows?

Reply:
Nothing :) I generally don't love to drop unknown data unless we have a real plan for it, but that ship has sailed a bit.  I'm fine dropping if you feel strongly about it.  it would be a good task to better understand the implications of dropping our unknown data at some point.

Reply:
No need to drop the data now. I'd like to run through the csv diff tool with you and see how far we are from having confidence making changes like that.